axiom/nr_log_events.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef NR_LOG_EVENTS_HDR
+#define NR_LOG_EVENTS_HDR
+
+#include "nr_analytics_events.h"
+#include "nr_log_event.h"
+#include "util_random.h"
+
+/*
+ * Purpose : Convert a log event to an analytic event and add it to
+ *           an event pool.
+ *
+ * Returns : true if sampling occured (see Notes below)
+ *           false otherwise
```

Comment:
Also false if the event was not added for some reason.

Reply:
Addressed in f426ac1f41bf07ec0bd24851078ce944edafd680